docs/sources/reference/components/discovery/discovery.relabel.md

```diff
@@ -78,7 +78,7 @@ In those cases, exported fields retain their last healthy values.
 
 ## Debug information
 
-`discovery.relabel` does not expose any component-specific debug information.
+`discovery.relabel` exposes some target-level debug information per target:
```

Comment:
```suggestion
`discovery.relabel` does not expose any component-specific debug information.
```
let's keep this one as it was to be consistent with the other components that also support live debugging